stackAndQueues/stacks/stacks.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+class Stack {
+  constructor() {
+    this._size = 0;
+  }
+
+  pop() {
+    if (!this._size()) {
```

Comment:
change `this._size()` to `this._size` because right now your code is reading it as a function when it is not. You also did in `peek()` and `isEmpty` function